client/src/common/components/CreationForm.jsx

```diff
@@ -97,7 +97,7 @@ class CreationForm extends PureComponent {
               type="submit"
               value={defaultName ? 'Update' : 'Create'}
             />
-            {type === CreationFormType.MODAL && (
+            {onCancel && (
```

Comment:
do we need nested condition here? What do you think about: {type === CreationFormType.MODAL && onCancel && (
)